src/tbp/monty/frameworks/loggers/graph_matching_loggers.py

```diff
@@ -255,6 +260,17 @@ def update_overall_stats(self, mode, episode, episode_steps, monty_matching_step
             )
             stats["monty_steps"].append(episode_steps)
             stats["monty_matching_steps"].append(monty_matching_steps)
+            # older LMs don't have prediction error stats
+            if "episode_avg_prediction_error" in episode_stats.keys():
+                stats["episode_avg_prediction_error"].append(
+                    episode_stats["episode_avg_prediction_error"]
+                )
+
+            stats["num_correct_child_or_parent"] += (
+                int(performance == "consistent_child_obj")
+                or int(performance == "correct")
+                or int(performance == "correct_mlh")
+            )
```

Comment:
This code seems too clever. Since these values are mutually exclusive, then the only possible values we're adding to this stat is 1 or 0. A better way to write this would be something like:
```suggestion
            if performance in {"consistent_child_obj", "correct", "correct_mlh"}:
                stats["num_correct_child_or_parent"] += 1
```

We could also use a list here by changing the `{}` to `[]`, but I think a set is faster for membership checks like this.